tests/features/vulnerability-explorer.feature

```diff
@@ -0,0 +1,71 @@
+    Then The Vulnerability Explorer page should display
+
+Scenario: Navigating to the Vulnerability Explorer page by searching for it in the search bar
+    Given User is on the Home page
+    When User clicks on the Search textbox
+    And User enters an ID of a Vulnerability
+    And User searches for the Vulnerability
+    And User selects the Vulnerability returned by the search
+    Then The Vulnerability Explorer page should display
+
+Scenario: Navigating to the Vulnerability Explorer page by filtering vulnerabilities in the list of all vulnerabilities
+    Given User is on the Home page
+    When User clicks on the Search button in the menu
+    And User selects the Vulnerabilities tab
+    And User selects a filter
+    And User selects an Vulnerability returned by the filter
+    Then The Vulnerability Explorer page should display
+
+Scenario: Display an overview of a vulnerability
+    Given User is on the Home page
+    When User navigates to the Vulnerabilities Explorer page
+    Then CVE name, CVSS score and CVE description should be visible
+    And "Show more" button is visible for vulnerabilities with a long descriptions
+    And Vulnerability metadata Reserved, Last Published Date and Last Modified should be visible
+    And Download button should be visible
+
+Scenario: Download a vulnerability CVE from the Vulnerability Explorer page
+    Given User is on the Home page
+    When User navigates to the Vulnerabilities Explorer page
+    And User clicks the Download button
+    Then The vulnerability CVE should download as a JSON file
+
+# Vulnerability-related Products
+Scenario: Display a list of products related to a single vulnerability
+    Given User is on the Home page
+    When User navigates to the Vulnerabilities Explorer page
+    And User clicks on the Related Products tab
+    Then The Name, Version, Status, Dependencies, Supplier and Created On data is visible for each product
+    And The name of each product should be a link to the overview of a corresponding SBOM Overview page
+    And The dependencies column should be expandable
+
+Scenario: Display a list of dependencies of a product related to a single vulnerability
+    Given User is on the Home page
+    When User navigates to the Vulnerabilities Explorer page
+    And User clicks on the Related Products tab
+    And User clicks on the Dependencies column of a product
+    Then The Type, Namespace, Name, Version, Path, Qualifiers and Dependency tree position data should display for each dependency
+
+# Vulnerability-related Advisories
+Scenario: Display advisories tied to a single vulnerability
+    Given User is on the Home page
+    When User navigates to the Vulnerabilities Explorer page
+    And User clicks on the Related Advisories tab
+    Then The ID, Title, Aggregated severity, Revision and Vulnerabilities data should display for each advisory tied to the vulnerability
+    And The ID should be a link a corresponding Advisory Overview page
```

Comment:
link `to` advisory

Reply:
@vobratil Can you please incorporate this minor change?

Reply:
Yes, sorry, I must have missed it. Done.